src/components/templates/components/Avatar.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+  <div>
+    <img
+      v-if="image"
```

Comment:
props not exist

Reply:
Thank you. This helped and I've fixed the issue now.